lintro/tools/core/install_strategies/cargo_strategy.py

```
@@ -0,0 +1,86 @@
+    def install_type(self) -> str:
+        """Return ``'cargo'``."""
+        return "cargo"
+
+    def is_available(self, env: InstallEnvironment) -> bool:
+        """Available if cargo is on PATH."""
+        return env.has("cargo")
+
+    def check_prerequisites(
+        self,
+        env: InstallEnvironment,
+        tool_name: str,
```

Ruff (0.15.7)

[warning] Unused method argument: `tool_name`

(ARG002)